src/layouts/Authentication/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+
+const layoutClassName = 'layout-authentication';
+
+export const layoutAuthenticationTestIds = {
+  base: layoutClassName,
+};
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+const LayoutAuthentication = ({ children }: LayoutProps) => {
+  useEffect(() => {
+    document.body.classList.add(layoutClassName);
+  });
```

Comment:
Should we also pass an empty array here to avoid unnecessary rerun? more info can be found [here](https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects)

```suggestion
  }, []);
```

Reply:
I have no idea with that but it's OK to me to add it https://github.com/Lahphim/react-surveyor-challenge/pull/55/commits/d1ef58937bf588cbf41688c01826843c5d6d8e4b